packs/src/races/half-elf.json

```
@@ -1,85 +1,163 @@
       "chat": "",
       "unidentified": ""
     },
-    "source": "SRD 5.1",
-    "activation": {
-      "type": "",
-      "cost": null,
-      "condition": ""
-    },
-    "duration": {
-      "value": "",
-      "units": ""
-    },
-    "cover": null,
-    "target": {
-      "value": null,
-      "width": null,
-      "units": "",
-      "type": ""
-    },
-    "range": {
-      "value": null,
-      "long": null,
-      "units": ""
-    },
-    "uses": {
-      "value": null,
-      "max": "",
-      "per": null,
-      "recovery": ""
-    },
-    "consume": {
-      "type": "",
-      "target": null,
-      "amount": null
-    },
-    "ability": null,
-    "actionType": "",
-    "attackBonus": "",
-    "chatFlavor": "",
-    "critical": {
-      "threshold": null,
-      "damage": ""
+    "source": {
+      "custom": "SRD 5.1",
+      "book": "",
+      "page": "",
+      "uuid": "",
+      "license": ""
     },
-    "damage": {
-      "parts": [],
-      "versatile": ""
+    "identifier": "half-elf",
+    "advancement": [
+      {
+        "_id": "rXyxzoqxdxlspMi5",
+        "type": "ItemGrant",
+        "configuration": {
+          "items": [
+            "Compendium.dnd5e.races.cnTbpPPeGW7vGjOV"
+          ],
+          "optional": false,
+          "spell": {
+            "ability": "",
+            "preparation": "",
+            "uses": {
+              "max": "",
+              "per": ""
+            }
+          }
+        },
+        "value": {},
+        "level": 0,
+        "title": ""
+      },
+      {
+        "_id": "Z9hvZFkWUNvowbQX",
+        "type": "AbilityScoreImprovement",
+        "configuration": {
+          "points": 1,
+          "fixed": {
+            "str": 0,
+            "dex": 0,
+            "con": 0,
+            "int": 0,
+            "wis": 0,
+            "cha": 2
+          },
+          "cap": 2
+        },
```

Half-elves get +2 Cha / +1 / +1

```suggestion
        "configuration": {
          "points": 2,
          "fixed": {
            "str": 0,
            "dex": 0,
            "con": 0,
            "int": 0,
            "wis": 0,
            "cha": 2
          },
          "cap": 1
        },
```